CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Added `Realm.subscriptions()` to query active query-based sync subscriptions. ([#2052](https://github.com/realm/realm-js/issues/2052))
+* Added `Realm.unsubscribe()` to unsubscribe ny name an active query-based sync subscriptions. ([#2052](https://github.com/realm/realm-js/issues/2052))
```

Comment:
`ny` ?

Reply:
Damn, `b` is too close to `n` on my keyboard.